src/libs/ReportActionComposeFocusManager.ts

```diff
@@ -18,7 +18,7 @@ let mainComposerFocusCallback: FocusCallback | null = null;
  *
  * @param callback callback to register
  */
-function onComposerFocus(callback: FocusCallback, isMainComposer = false) {
+function onComposerFocus(callback: FocusCallback | null, isMainComposer = false) {
```

Comment:
With the suggested changes above, this change is not needed.

Reply:
Seems like this is resolved now.